Modules/socketmodule.c

```diff
@@ -518,6 +518,15 @@ remove_unusable_flags(PyObject *m)
 #endif
 #endif
 
+#if defined(MS_WINDOWS) && defined(AF_BTH)
```

Comment:
Looks like `AF_BTH` is defined in all supported versions of Windows, so no need for the second part of the test.

Reply:
Done -- only checks for `MS_WINDOWS` now.